app/services/calculation_orchestrator.rb

```diff
@@ -1,19 +1,18 @@
 # frozen_string_literal: true
 
-require "participant_event_aggregator"
 require "payment_calculator/ecf/payment_calculation"
 
 class CalculationOrchestrator
   class << self
     def call(cpd_lead_provider:,
              contract:,
-             aggregator: ::ParticipantEventAggregator,
+             recorder: ParticipantDeclaration::ECF,
```

Comment:
Even though we've remove the proxy, I'd still have called it the aggregator, since it's acting like an aggregator for the orchestrator.

Reply:
Hmm, alright, I went for `recorder` cause that is what it was called in the old aggregator :)